grails-app/i18n/messages_en.properties

```diff
@@ -236,6 +236,7 @@ list.facetcheckboxes.button.resetfacetoptions = Reset to defaults
 list.facetcheckboxes.div02.title = Custom facets
 list.resultsretuened.span.returnedtotal = records returned of
```

Comment:
These i18n codes have a typo - `resultsretuened` should be `resultsreturned`

Reply:
Hi Nick, there's more than 1 place that has this typo:

list.**resultsretuened**.span.returnedtotal = records returned of
list.**resultsretuened**.span.returnedtext = for
list.**resultsretuened**.span.returnedtext1 = results for
list.**resultsretuened**.click.to.remove.filters = Click to remove this filter
list.**resultsretuened**.spatial.filter = Spatial filter
list.**resultsretuened**.circle = CIRCLE
list.**resultsretuened**.button01 = Clear all
list.**resultsretuened**.button01.title = Click to clear all filters
list.**resultsretuened**.navigator01 = placeholder
list.**resultsretuened**.navigator01.title = view species page
list.**resultsretuened**.click.more.info = click for more info on this query
list.**resultsretuened**.div01.des01 = The search results include records for synonyms and child taxa of
list.**resultsretuened**.div01.des02 = placeholder
list.**resultsretuened**.div01.des03 = link placeholder
list.**resultsretuened**.form.des01 = The result set contains records provided under the following names
list.**resultsretuened**.form.button01 = Refine search
list.**resultsretuened**.restrict.results = Restrict results to the selected names
list.**resultsretuened**.form.div01 = placeholder taxa list

should I

1. change all occurrences of this typo or just the new one I created in this fix
2. If to change all, do I need to change typos in message files of other languages?